src/components/CopyButton/CopyButton.tsx

```diff
@@ -0,0 +1,56 @@
+import { useMatomo } from '@datapunt/matomo-tracker-react';
+import React, { useState } from 'react';
+import { Button } from 'react-bootstrap';
+import { toast } from 'react-toastify';
+
+interface Props {
+    onCopy?(),
+    successMessage?: JSX.Element,
+    copyValue: string
+    buttonWrapperClass?: string,
+    buttonVariant?: string
+}
+
+export function CopyButton(props: Props) {
+
+    let [isCopied, setIsCopied] = useState(false);
+    let { trackEvent } = useMatomo();
+
+    let copyIcon = (
+        <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" className="bi bi-clipboard" viewBox="0 0 16 16">
+            <path d="M4 1.5H3a2 2 0 0 0-2 2V14a2 2 0 0 0 2 2h10a2 2 0 0 0 2-2V3.5a2 2 0 0 0-2-2h-1v1h1a1 1 0 0 1 1 1V14a1 1 0 0 1-1 1H3a1 1 0 0 1-1-1V3.5a1 1 0 0 1 1-1h1v-1z" />
+            <path d="M9.5 1a.5.5 0 0 1 .5.5v1a.5.5 0 0 1-.5.5h-3a.5.5 0 0 1-.5-.5v-1a.5.5 0 0 1 .5-.5h3zm-3-1A1.5 1.5 0 0 0 5 1.5v1A1.5 1.5 0 0 0 6.5 4h3A1.5 1.5 0 0 0 11 2.5v-1A1.5 1.5 0 0 0 9.5 0h-3z" />
+        </svg>
+    );
+
+    let copiedIcon = (
+        <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" className="bi bi-clipboard-check" viewBox="0 0 16 16">
+            <path fillRule="evenodd" d="M10.854 7.146a.5.5 0 0 1 0 .708l-3 3a.5.5 0 0 1-.708 0l-1.5-1.5a.5.5 0 1 1 .708-.708L7.5 9.793l2.646-2.647a.5.5 0 0 1 .708 0z" />
+            <path d="M4 1.5H3a2 2 0 0 0-2 2V14a2 2 0 0 0 2 2h10a2 2 0 0 0 2-2V3.5a2 2 0 0 0-2-2h-1v1h1a1 1 0 0 1 1 1V14a1 1 0 0 1-1 1H3a1 1 0 0 1-1-1V3.5a1 1 0 0 1 1-1h1v-1z" />
+            <path d="M9.5 1a.5.5 0 0 1 .5.5v1a.5.5 0 0 1-.5.5h-3a.5.5 0 0 1-.5-.5v-1a.5.5 0 0 1 .5-.5h3zm-3-1A1.5 1.5 0 0 0 5 1.5v1A1.5 1.5 0 0 0 6.5 4h3A1.5 1.5 0 0 0 11 2.5v-1A1.5 1.5 0 0 0 9.5 0h-3z" />
+        </svg>
+    );
+
+    function copyClick() {
+        setIsCopied(true);
+        window.navigator.clipboard.writeText(props.copyValue);
+        if (props.onCopy) {
+            props.onCopy();
+        }
+        if (props.successMessage) {
+            toast.success(props.successMessage)
+        }
+        trackEvent({
+            category: 'copyButtonClick',
+            action: props.copyValue
+        })
+    }
+
+    return (
+        <div>{window.navigator.clipboard ?
+            <div className={props.buttonWrapperClass}>
+                <Button variant={props.buttonVariant || "secondary"} onClick={() => { copyClick() }}>{isCopied ? copiedIcon : copyIcon}</Button>
```

Comment:
Button should have attribute `aria-label="copy to clipboard"`
See https://web.dev/button-name/